src/main/fc/init.c

```diff
@@ -295,6 +295,12 @@ void init(void)
     }
 #endif
 
+    systemClockSetHSEValue(systemConfig()->hseMhz * 1000000U);
+
+#ifdef USE_OVERCLOCK
```

Comment:
Any reason that this was moved? I believe the Spektrum bind facility was moved to where it was now to get the timing right, so if there is no compelling reason to move things around they should probably be left where they are now.

Reply:
@mikeller Da** You are right about keeping the position of the code, but for a different reason; the new clock source code will cause MCU to reboot, which will prevent the binding procedure to execute as it has a logic to execute only once after hard reset.

The code was tested when the new clock code didn't soft reset after new clock source is being set. The code was changed to soft reset to eliminate a need for an extra manual reboot.

Reply:
I'll add a comment for the placement.